web/src/components/Content.tsx

```diff
@@ -44,6 +44,9 @@ export const Content = (props: ContentProps): ReactElement => {
       return <HorizontalLine />;
     },
     blockquote: GreenBox,
+    greenBox: (props: any): ReactElement => {
```

Comment:
Is there a particulalry easy way to check that this is working in the app? I was gonna let content test this out on their own.

I'm curious about saving changes in the CMS locally and seeing them locally but I'm not super confident in how to do that at the moment without screwing anything up. 

Like in the localhost version of the CMS can I just publish new changes and see them only on local or do I need to change the branch in my local config.yml first or something else entirley? Would love a bit more clarity if a reviewer has the answers here.

Reply:
@adidner I don't know how to do this locally. If you wanted to see this rendered, you could just manually modify one of the `md` files and add the syntax.
```
:::greenBox 
 This is a test of the emergency alert system.
:::
```

It does seem to be working nicely though. I accidentally [created a PR](https://github.com/newjersey/navigator.business.nj.gov/pull/6131/files) with the change.

Reply:
Thats interesting, I'm gonna try and re-produce locally

Reply:
It might not be working because of what we said regarding the CMS content pointing at the content branch

Reply:
Works for me when I run the proxy server